.pre-commit-config.yaml

```diff
@@ -1,9 +1,8 @@
 repos:
-- repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.22.0
-  hooks:
-    - id: terraform_fmt
-    - id: terraform_docs
-      args: [--args=--with-aggregate-type-defaults --no-escape]
```

Comment:
We need these args but please do update the version 🙂

Reply:
ok, any reason why the terraform-dcos is failing? I would also run the tests, but running into ruby version hell. I need to get rvm going.